utils/accessibility_util.py

```diff
@@ -0,0 +1,21 @@
+"""
+This is a class object that contains the following methods
```

Comment:
A minor change needs to be made to the module doc-string here, since we have removed the `class`, can you replace it with `module`

Reply:
Done!